templates/jobs/cf-deploy.groovy.j2

```diff
@@ -67,7 +67,12 @@ echo Retrieving Terraform state
 ln -sf ~/${DEPLOY_ENV}_{{ platform }}.tfstate {{ platform }}/${DEPLOY_ENV}.tfstate
 ln -sf ~/${DEPLOY_ENV}_{{ platform }}.tfstate.backup {{ platform }}/${DEPLOY_ENV}.tfstate.backup
 
-make {{ platform }} DEPLOY_ENV=${DEPLOY_ENV} ROOT_PASS_DIR=jenkins
+{% if deploy_trial_ssl_certificates is defined %}
+  make {{ platform }}-trial DEPLOY_ENV=${DEPLOY_ENV} ROOT_PASS_DIR=jenkins
```

Comment:
The indentation has introduced some extra whitespace in the job config. While the template logically looks better with the indentation, it might be better to not have the whitespace in the job config